### PR TITLE
[libtorrent] Fix build on arm-linux community triplet

### PR DESCRIPTION
**Pull request by FranciscoPombal**
Tested on a Raspberry Pi 4 running the latest stable release of Raspbian/Raspberry PI OS (32-bit).

No additional features were tested, just the basic baseline package.

### CODE REVIEW

**Comment by FranciscoPombal** [comment]
CI failures are unrelated.

File: ports/libtorrent/CONTROL [hunk 1]
```diff
@@ -2,7 +2,7 @@ Source: libtorrent
 Version: 1.2.10
```
Comment (NancyLi1013) [inline ports/libtorrent/CONTROL:2]:
```suggestion
Version: 1.2.10
Port-Version: 1
```

**Review by NancyLi1013 — CHANGES_REQUESTED** [review]

**Comment by NancyLi1013** [comment]
Hi @FranciscoPombal 
Thanks for this PR.
Have you tested if these features can work fine?

**Comment by FranciscoPombal** [comment]
@NancyLi1013 

> Have you tested if these features can work fine?

I haven't tested any additional features, I fixed the baseline installation: `vcpkg install libtorrent:arm-linux`. In the future I will try out the features and submit follow-up PRs (or an issue report) if needed.

**Review by NancyLi1013 — APPROVED** [review]

File: ports/libtorrent/CONTROL [hunk 2]
```diff
@@ -1,8 +1,9 @@
 Source: libtorrent
 Version: 1.2.10
+Port-Version: 1
 Homepage: https://github.com/arvidn/libtorrent
 Description: An efficient feature complete C++ BitTorrent implementation
-Build-Depends: openssl, boost-system, boost-date-time, boost-chrono, boost-random, boost-asio, boost-crc, boost-config, boost-iterator, boost-scope-exit, boost-multiprecision, boost-variant
+Build-Depends: openssl, boost-system, boost-date-time, boost-chrono, boost-random, boost-asio, boost-crc, boost-config, boost-iterator, boost-scope-exit, boost-multiprecision, boost-pool (linux&arm), boost-variant
```
Comment (ras0219-msft) [inline ports/libtorrent/CONTROL:6]:
```suggestion
Build-Depends: openssl, boost-system, boost-date-time, boost-chrono, boost-random, boost-asio, boost-crc, boost-config, boost-iterator, boost-scope-exit, boost-multiprecision, boost-pool, boost-variant
```
Based on my reading of the code, this is always used, so it should always be directly depended upon.

**Review by ras0219-msft — CHANGES_REQUESTED** [review]
LGTM once my suggestion is applied.

Thanks for the PR!

File: ports/libtorrent/CONTROL (hunk 2 above)
Comment (FranciscoPombal) [reply ports/libtorrent/CONTROL:6]: @ras0219 

You're right. But for some reason it is only required to be explicitly specified as a dependency with `linux&arm`. I guess on other triplets this was working by accident all along.

Comment (ras0219) [reply ports/libtorrent/CONTROL]: Yep; the other triplets were getting it transitively through one of the other boost dependencies.

**Comment by ras0219-msft** [comment]
Thanks again!

**Pull request merged**